src/js/components/__tests__/__snapshots__/components-test.js.snap

```diff
@@ -1431,7 +1431,7 @@ selectors",
       },
     ],
     "description": "A graphical chart.",
-    "name": "Chart",
+    "name": undefined,
```

Comment:
Probably has the same root cause as the README changes

Reply:
fixed